viewer/src/self-contained.ts

```diff
@@ -1,12 +1,21 @@
 import Engine from "@gaia-project/engine";
+import { FactionVariant } from "@gaia-project/engine/src/engine";
```

Comment:
Last comment ^^

Export that from `engine`, so you can do `import Engine, { FactionVariant} from "@gaia-project/engine"`